_posts/2015-03-stitch-fix-heart-unix.md

```diff
@@ -0,0 +1,139 @@
+*Proposed Outline:*
+```
+I. Introduction
+II. Table of contents by problem
+III. Each problem
+  A. Title
+    1. Attribution and environment sub-heading
+  B. Problem
+  C. Solution
+    1. Input
+    2. Command
+    3. Explanation
+IV. ~~Closing~~?
```

Comment:
Since the post is a list, a *Closing* doesn't seem natural.  Maybe the *Introduction* has a nice closing sentence (or sentences) to wrap up the post?